content/_code-samples/issue-a-token/js/list-account-tokens.js

```diff
@@ -0,0 +1,59 @@
+if (typeof module !== "undefined") {
+    // Use var here because const/let are block-scoped to the if statement.
+    var xrpl = require('xrpl')
+  }
+  // Show tokens and other objects owned by an account.
+  // https://xrpl.org/account_objects.html#account_objects
+  // https://xrpl.org/account_lines.html#account_lines
+
+  async function main() {
+    // Testnet example: r9CEVt4Cmcjt68ME6GKyhf2DyEGo2rG8AW
+    account = "r9CEVt4Cmcjt68ME6GKyhf2DyEGo2rG8AW"
+
+    // Connect to a testnet node
+    console.log("Connecting to Testnet...")
+    const client = new xrpl.Client('wss://s.altnet.rippletest.net:51233/')
+    await client.connect()
+
+    // Query the ledger for an account's objects and its trustlines...
+    const response_objects = await client.request({
+        "command": "account_objects",
+        "account": account,
+        "ledger_index": "validated",
+    })
+
+    const response_lines = await client.request({
+        "command": "account_lines",
+        "account": account,
+        "ledger_index": "validated",
```

Comment:
```suggestion
        command: "account_lines",
        account: account,
        ledger_index: "validated",
```
Strings aren't required for transaction keys. Can you refactor it everywhere else?